cards/time-lord.md

```diff
@@ -0,0 +1,11 @@
+# Time Lord
+
+### Type: Defensive
+### Time of Play: Any
+
+### Card Intent & Rules:
+The player who made the play resulting in this card being drawn is given the title of **_Time Lord_**. The **_Time Lord_** has the ability to roll back works like the _Time Warp_ expect is indefinite. If the **_Time Lord_** is the server when his/her team loses he can continue serving and roll back the last point until his team scores. This card is indefinite throughout the game so each time the **_Time Lord_** rotates in to serve this power is active.
```

Comment:
@afaur You can tell we've been writing tests too long when we're adding "expect is definite" XD